ci-operator/step-registry/openshift/e2e/openstack/techpreview/openshift-e2e-openstack-techpreview-workflow.yaml

```diff
@@ -0,0 +1,16 @@
+workflow:
+  as: openshift-e2e-openstack-techpreview
+  steps:
+    pre:
+    - chain: ipi-openstack-pre-techpreview
+    test:
+      - ref: openshift-e2e-test
+    post:
+    - chain: ipi-openstack-post
+    env:
+      CONFIG_TYPE: "minimal"
+      BASE_DOMAIN: shiftstack.devcluster.openshift.com
+  documentation: |-
+    The openshift-e2e-openstack-techpreview workflow executes the common
```

Comment:
No serial test-suite for OpenStack?

Reply:
For openstack, this appears to be overriden by an environment variable within the job configuration itself. I'm just trying to be consistent with existing openstack jobs by copying that pattern